components/speedreader/resources/panel/components/options-control/index.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+
+import * as S from './style'
+import ThemeControl from '../theme-control'
+
+import { AppearanceSettings, Theme, FontSize, FontFamily } from '../../api/browser'
+import { FontStyleList, FontSizeList } from '../lists'
+import { getLocale } from '$web-common/locale'
+
+interface AppearanceControlProps {
+  appearanceSettings: AppearanceSettings
+  onThemeChange: (theme: Theme) => void
+  onFontFamilyChange: (fontFamily: FontFamily) => void
+  onFontSizeChange: (fontSize: FontSize) => void
+  onClose: () => void
+}
+
+function AppearanceControl (props: AppearanceControlProps) {
+  return (
+    <S.Box>
+      <ThemeControl
+        activeTheme={props.appearanceSettings.theme}
+        onClick={props.onThemeChange}
+      />
+      <S.VDelemiter />
```

Comment:
These `VDelemiter` elements look redundant and add noise to the markup. Can we achieve the same effect by using a `div` wrapped around with a `border-right` style, or by setting `border-right` on the parent for all `div` elements except the last child? This would help simplify the markup and reduce unnecessary elements.